components/org.wso2.carbon.appmgt.rest.api.publisher/src/main/java/org/wso2/carbon/appmgt/rest/api/publisher/impl/AdministrationApiServiceImpl.java

```diff
@@ -23,44 +23,230 @@
 import org.apache.commons.logging.LogFactory;
 import org.wso2.carbon.appmgt.api.APIProvider;
 import org.wso2.carbon.appmgt.api.AppManagementException;
+import org.wso2.carbon.appmgt.api.model.BusinessOwner;
+import org.wso2.carbon.appmgt.api.model.BusinessOwnerProperty;
 import org.wso2.carbon.appmgt.api.model.entitlement.EntitlementPolicyPartial;
 import org.wso2.carbon.appmgt.rest.api.publisher.AdministrationApiService;
 import org.wso2.carbon.appmgt.rest.api.publisher.dto.BusinessOwnerDTO;
+import org.wso2.carbon.appmgt.rest.api.publisher.dto.BusinessOwnerListDTO;
+import org.wso2.carbon.appmgt.rest.api.publisher.dto.BusinessOwnerPropertiesDTO;
 import org.wso2.carbon.appmgt.rest.api.publisher.dto.PolicyPartialDTO;
 import org.wso2.carbon.appmgt.rest.api.util.dto.ErrorDTO;
 import org.wso2.carbon.appmgt.rest.api.util.exception.InternalServerErrorException;
 import org.wso2.carbon.appmgt.rest.api.util.utils.RestApiUtil;
 import org.wso2.carbon.appmgt.rest.api.util.validation.BeanValidator;
+import org.wso2.carbon.utils.xml.StringUtils;
 
 import javax.ws.rs.core.Response;
+import java.util.ArrayList;
+import java.util.List;
 
 public class AdministrationApiServiceImpl extends AdministrationApiService {
     private static final Log log = LogFactory.getLog(AdministrationApiServiceImpl.class);
     BeanValidator beanValidator;
 
     @Override
     public Response administrationBusinessownerGet(String accept, String ifNoneMatch) {
-        return null;
+        BusinessOwnerListDTO businessOwnerListDTO = new BusinessOwnerListDTO();
+        try {
+            APIProvider apiProvider = RestApiUtil.getLoggedInUserProvider();
+            //get business owner related to id.
+            List<BusinessOwnerDTO> businessOwnerDTOList = new ArrayList<>();
+            List<BusinessOwner> businessOwners = apiProvider.getBusinessOwners();
+            List<BusinessOwnerProperty> businessOwnerPropertyList;
+            if (businessOwners.isEmpty()) {
+                return RestApiUtil.buildNotFoundException("Business Owners", null).getResponse();
+            }
+
+            for (BusinessOwner businessOwner : businessOwners) {
+                BusinessOwnerDTO businessOwnerDTO = new BusinessOwnerDTO();
+                businessOwnerDTO.setId(businessOwner.getBusinessOwnerId());
+                businessOwnerDTO.setName(businessOwner.getBusinessOwnerName());
+                businessOwnerDTO.setEmail(businessOwner.getBusinessOwnerEmail());
+                businessOwnerDTO.setDescription(businessOwner.getBusinessOwnerDescription());
+                businessOwnerDTO.setSite(businessOwner.getBusinessOwnerSite());
+                businessOwnerPropertyList = businessOwner.getBusinessOwnerPropertiesList();
+                List<BusinessOwnerPropertiesDTO> businessOwnerPropertiesDTOList = new ArrayList<>();
+                //save custom properties of the owner.
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/0d9719587a489bf2ef909c862c46bdbce01f1af0